week-1-practise/level-1/7-logic-error-4.js

```diff
@@ -1,11 +1,11 @@
 // The syntax for this function is valid but it has an error, find it and fix it.
 
 function multiply(a, b, c) {
-  a * b * c;
-  return;
+  
+  return a * b * c;
 }
 
-var result = multiply(1, 3, 4);
+const result = multiply(1, 3, 4);
```

Comment:
Nice one to recognize that this can be a `const`

Reply:
👍